### ISSUE TITLE
Perhaps refactor papirus-temp with device tree driver for LM75

### ISSUE THREAD

**Issue by shawaj** [PiSupply/PaPiRus#85]
Perhaps we could look at using the newly created LM75 temp sensor device tree driver and update the papirus-temp script accordingly?

In future, this could be incorporated as a device tree blob onto the EEPROM on the PaPiRus HAT

**Comment by shawaj** [comment]
Take a look here - https://www.raspberrypi.org/forums/viewtopic.php?p=1130515#p1130515

And here:

https://github.com/raspberrypi/linux/search?utf8=✓&q=Lm75

**Comment by shawaj** [comment]
Also see #41 

Would be good to change this to a python script rather than a bash script (could keep the bash script as a legacy script somewhere, as seems pointless to just flat out delete it but worth having more continuity in programming language used)

**Comment by francesco-vannini** [comment]
@tvoverbeek is this taken care of in #117 ?

**Comment by tvoverbeek** [comment]
No. The I2C interface for the LM75 (as long as you are only interested in the temperature) is very simple. Does not warrant to use the device-tree for this simple interface.
Converting it to a python script is something worth doing.

**Comment by shawaj** [comment]
The device tree driver is already there I believe? Just need to add a line to the config.txt I think to enable it

**Comment by tvoverbeek** [comment]
Yes, but when enabled you cannot access it directly via I2C in python using smbus.. Would have to rewrite the LM75B access in lm75.py (imported by epd.py) to make use of the kernel driver interface. Moreover installation then requires checking of config.txt for the correct dtoverlay.

**Comment by shawaj** [comment]
@tvoverbeek cool - lets park this one for now then. Seems like a bad idea!
